src/GraphQL/Types/Schema.cs

```diff
@@ -125,7 +125,7 @@ public Schema(IServiceProvider services, IEnumerable<IConfigureSchema> configura
             _services = services;
 
             Directives = new SchemaDirectives();
-            Directives.Register(Directives.Include, Directives.Skip, Directives.Deprecated);
+            Directives.Register(Directives.Include, Directives.Skip, Features.DeprecationOfInputValues ? Directives.DeprecatedDraft : Directives.Deprecated);
```

Comment:
Is `Features` a static property?

Reply:
Ok then it is impossible to initialize the proper directives with this code because `Features` cannot be accessed prior to the constructor being executed.  See comments above.

Reply:
Fixed. Overlooked it.